vl-convert-python/vl_convert.pyi

```diff
@@ -0,0 +1,801 @@
+    ...
+
+def vega_to_png(
+    vg_spec: VlSpec,
+    scale: float,
+    ppi: float | None,
+    allowed_base_urls: list[str] | None,
+    format_locale: FormatLocale | None,
+    time_format_locale: TimeFormatLocale | None,
+) -> bytes:
+    """
+    Convert a Vega spec to PNG image data.
+
+    Parameters
+    ----------
+    vg_spec
+        Vega JSON specification string or dict
+    scale
+        Image scale factor (default 1.0)
+    ppi
+        Pixels per inch (default 72)
+    allowed_base_urls
+        List of allowed base URLs for external data requests.
+        Default allows any base URL
+    format_locale
+        d3-format locale name or dictionary
+    time_format_locale
+        d3-time-format locale name or dictionary
+
+    Returns
+    -------
+    PNG image data.
+    """
+    ...
+
+def vega_to_scenegraph(
+    vg_spec: VlSpec,
+    allowed_base_urls: list[str] | None,
+    format_locale: FormatLocale | None,
+    time_format_locale: TimeFormatLocale | None,
+) -> dict[str, Any]:
+    """
+    Convert a Vega spec to a Vega Scenegraph.
+
+    Parameters
+    ----------
+    vg_spec
+        Vega JSON specification string or dict
+    allowed_base_urls
+        List of allowed base URLs for external data requests.
+        Default allows any base URL
+    format_locale
+        d3-format locale name or dictionary
+    time_format_locale
+        d3-time-format locale name or dictionary
+
+    Returns
+    -------
+    scenegraph.
+    """
+    ...
+
+def vega_to_svg(
+    vg_spec: VlSpec,
+    allowed_base_urls: list[str] | None,
+    format_locale: FormatLocale | None,
+    time_format_locale: TimeFormatLocale | None,
+) -> str:
+    """
+    Convert a Vega spec to an SVG image string.
+
+    Parameters
+    ----------
+    vg_spec
+        Vega JSON specification string or dict
+    allowed_base_urls
+        List of allowed base URLs for external data requests.
+        Default allows any base URL
+    format_locale
+        d3-format locale name or dictionary
+    time_format_locale
+        d3-time-format locale name or dictionary
+
+    Returns
+    -------
+    SVG image string.
+    """
+    ...
+
+def vega_to_url(vg_spec: VlSpec, fullscreen: bool | None) -> str:
+    """
+    Convert a Vega spec to a URL that opens the chart in the Vega editor.
+
+    Parameters
+    ----------
+    vg_spec
+        Vega JSON specification string or dict
+    fullscreen
+        Whether to open the chart in full screen in the editor
+
+    Returns
+    -------
+    URL string.
+    """
+    ...
+
+def vegalite_to_html(
+    vl_spec: VlSpec,
+    vl_version: str | None,
+    bundle: bool | None,
+    config: dict[str, Any] | None,
+    theme: VegaThemes | None,
+    format_locale: FormatLocale | None,
+    time_format_locale: TimeFormatLocale | None,
+    renderer: Renderer | None,
+) -> str:
+    """
+    Convert a Vega-Lite spec to self-contained HTML document using a particular version of the Vega-Lite JavaScript library.

```

Comment:
I had to disable [line-too-long](https://docs.astral.sh/ruff/rules/line-too-long/) for docs like this.

Splitting it into 2 lines broke[multi-line-summary-second-line](https://docs.astral.sh/ruff/rules/multi-line-summary-second-line/). 

Just thought I'd note this with links to the rules, not proposing any changes